packages/dialog/README.md

```diff
@@ -0,0 +1,281 @@
+        <DialogContent>
+          <p>Discard Draft?</p>
+        </DialogContent>
+        <DialogFooter>
+          <DialogButton action='dismiss'>Cancel</DialogButton>
+          <DialogButton action='discard' isDefault>Discard</DialogButton>
+        </DialogFooter>
+      </Dialog>
+    );
+  }
+}
+
+
+```
+
+
+#### Simple Dialog
+
+The Simple Dialog contains a list of potential actions. It does not contain buttons.
+
+```js
+import React, {Component} from 'react';
+import Dialog, {DialogTitle, DialogContent} from '@material/react-dialog';
+import List, {ListItem, ListItemGraphic, ListItemText} from '@material/react-dialog';
+import MaterialIcon from '@material/react-material-icon';
+
+class Simple extends Component {
+  state = {
+    isOpen: true
+    choices: ['user1@example.com', 'user2@example.com', 'Add Account']
+    action: ''
+  };
+
+  render() {
+    return (
+      <Dialog 
+        open={isOpen} 
+        onClose={(action) => this.setState({action, isOpen: false})}>
+        <DialogTitle>Select User</DialogTitle>
+        <DialogContent>
+            <List avatarList>
+              {choices.map((choice, i) => (
+                <ListItem key={i} data-mdc-dialog-action={choice}>
+                  <ListItemGraphic graphic={
+                    <MaterialIcon icon={choice.match(/@/) ? 'person' : 'add'}/>
+                  }/>
+                  <ListItemText primaryText={choice}/>
+                </ListItem>
+              ))
+              }
+            </List>
+        </DialogContent>
+      </Dialog>
+    );
+  }
+}
+ ```
+>NOTES: (1) the inclusion of the `avatarList`, which aligns with the Simple
+>Dialog spec. (2) the `data-mdc-dialog-action` which singals the chosen action `onClose()`
+
+
+### Confirmation Dialog
+
+The Confirmation Dialog contains a list of choices, and buttons to confirm or
+cancel. Choices are accompanied by radio buttons (indicating single selection)
+or checkboxes (indicating multiple selection).
+
+```js
+import React, {Component} from 'react';
+import Dialog, {
+  DialogTitle,
+  DialogContent,
+  DialogFooter,
+  DialogButton,
+} from '@material/react-dialog';
+import List, {ListItem, ListItemText} from '@material/react-list';
+import Radio, {NativeRadioControl} from '@material/react-radio';
+
+
+class Confirmation extends Component
+  state = {
+    isOpen: false, 
+    action: '', 
+    selectedIndex: -1,
+    choices: ['Never gonna give you up', 'Host cross buns', 'None'];
+   };
+
+  isChecked = (i) => i === this.state.selectedIndex;
+
+  render() {
+    return (
+      <Dialog
+        onClose={(action) => this.setState({isOpen: false, action})}
+        open={this.state.isOpen}>
+        <DialogTitle>Chose a Phone Ringtone</DialogTitle>
+        <DialogContent>
+          <List 
+            singleSelection 
+            handleSelect={ (selectedIndex) => this.setState({selectedIndex})}
+          >{choices.map((choice, i) => {
+              let c = choice.replace(/\s/g, '-');
```

Comment:
can you make `c` a full variable name? We should have good naming practices in our readmes.